src/validations.js

```diff
@@ -0,0 +1,11 @@
+export function validateEmail(email) {
+  if (!email) {
+    return 'validation_messages.error_required_field';
+    // eslint-disable-next-line
```

Comment:
Why do you need to disable eslint validation?

Reply:
Oh, I see:

![image](https://user-images.githubusercontent.com/1157864/55919991-72449a80-5bce-11e9-9a03-eb2a127bb98a.png)

But, ESLint is right, those escapes are not necessary because, for example, `.` does not require escape when it is inside `[. . .]`.

This regex seems to be equivalent: `^([a-z\d[\]])([\w-.+\]\[]*)@([ñ\w.-\]\[]+)\.[\w-\]\[.]{2,}(\?)?.*$`

Reply:
Fixed